tests/foreman/endtoend/test_api_endtoend.py

```diff
@@ -849,7 +849,7 @@
         '/katello/api/sync_plans',
         '/katello/api/sync_plans/:id/sync',
     ),
-    'sync': ('/katello/api/organizations/:organization_id/products/:product_id/sync',),
+    'sync': ('/katello/api/repositories/:repository_id/sync',),
```

Comment:
I don't see product sync api here. Did we miss it here?

Reply:
Well, it was removed intentionally in 2159105, some more notes in https://github.com/Katello/katello/pull/10632 
I'm not owning that bz, so if the future qe contact of that bz negotiates different changes I'd be happy to adjust this test again

Reply:
I see only one of the endpoints in the BZ got changed in the end, the other two were kept. Is that intended?

Reply:
@shweta83 the removed endpoint was about monitoring the sync status, product sync api is still there under  `/katello/api/products/:id/sync`

Reply:
@lhellebr hm, you're right, I'll check with the QA owner of that bz